addons/explosives/functions/fnc_addClacker.sqf

```diff
@@ -34,13 +34,13 @@ private _detonators = [_unit] call FUNC(getDetonators);
 } forEach _requiredItems;
 
 if !(_hasRequired) exitWith {};
-private _config = configFile >> "CfgMagazines" >> _magazineClass >> "ACE_Triggers" >> configName _config;
+private _triggerConfig = configFile >> "CfgMagazines" >> _magazineClass >> "ACE_Triggers" >> configName _config;
 
 private _clacker = _unit getVariable [QGVAR(Clackers), []];
 GVAR(PlacedCount) = GVAR(PlacedCount) + 1;
 
-_clacker pushBack [_explosive, getNumber(_config >> "FuseTime"), format [localize LSTRING(DetonateCode),
-    GVAR(PlacedCount)], _magazineClass, configName ((_this select 3) select (count (_this select 3) - 1))];
+_clacker pushBack [_explosive, getNumber(_triggerConfig >> "FuseTime"), format [localize LSTRING(DetonateCode),
+    GVAR(PlacedCount)], _magazineClass, configName _config];
```

Comment:
```suggestion
    GVAR(PlacedCount)], _magazineClass, configName _triggerConfig];
```

FYI I'm working on a cleanup of explosives, which handles improving stuff like this.

Reply:
`_config` used to be shadowed on line 37 but I changed it, so I think this is correct

if these changes are just going to cause merge conflicts I'm happy to drop it

Reply:
> `_config` used to be shadowed on line 37 but I changed it, so I think this is correct

You are correct, my bad.

> if these changes are just going to cause merge conflicts I'm happy to drop it

Probably best - I'll be making the PR soonTM.